fastify.d.ts

```diff
@@ -119,7 +119,7 @@ type TrustProxyFunction = (address: string, hop: number) => boolean
 /* Export all additional types */
 export { FastifyRequest, RequestGenericInterface } from './types/request'
 export { FastifyReply } from './types/reply'
-export { FastifyPluginCallback, FastifyPluginAsync, FastifyPluginOptions, FastifyPlugin } from './types/plugin'
+export { FastifyPluginOptions, FastifyPlugin } from './types/plugin'
```

Comment:
@mcollina, @fox1t this breaks `fastify-plugin`.
Also it's the breaking change cause it affects public APIs for TS users :(

![image](https://user-images.githubusercontent.com/11780854/93080497-f8b0b600-f6b7-11ea-8370-14a966909dbd.png)

Reply:
I'm not sure I understand the message (it suggests to import the same thing), but could you verify that you are using most recent `fastify-plugin` version? (it should be fixed in 2.0.1 ->https://github.com/fastify/fastify-plugin/commit/d81e6a6b32effc6607e2e7b58c4365d1ec60dc61)

Reply:
I use 2.3.4

Reply:
It was released 4 hours ago

Reply:
Oh, now I see the issue.
I don't know how these got removed. I'll put a PR to get them back

Reply:
Thanks!

Reply:
Done https://github.com/fastify/fastify/pull/2555.